com.microsoft.mrtk.graphicstools.unity/Editor/ShaderGUIs/StandardShaderGUI.cs

```diff
@@ -626,7 +629,9 @@ protected void MainMapOptions(MaterialEditor materialEditor, Material material)
 
             if (PropertyEnabled(enableEmission))
             {
-                materialEditor.ShaderProperty(emissiveColor, Styles.emissiveColor, 2);
+                EditorGUI.indentLevel += 2;
+                materialEditor.TexturePropertySingleLine(Styles.emissiveColor,emissiveMap, emissiveColor);

```

Comment:
nitpick: missing a space:
```suggestion
                materialEditor.TexturePropertySingleLine(Styles.emissiveColor, emissiveMap, emissiveColor);

```